app/views/pseuds/index.html.erb

```diff
@@ -11,6 +11,10 @@
 <% end %>
 <!--/subnav-->
 
+<!--subnav-->
+<%= will_paginate @pseuds %>
```

Comment:
Wait, sorry, one more comment -- can you move `<%= will_paginate @pseuds %>` up above the `<!--/subnav-->` on line 12? We usually just have one subnav comment on the top and one on the bottom.